### ISSUE TITLE
Travel cost calculator

### ISSUE THREAD

**Issue by BartKieviet** [Tsunder/Pardus-Sweetener#68]
We have the planner within the sector, let's take it to the next level and make a universe travel cost calculator from our current location to one picked on a map.

ToDo list
- [ ] Rewrite `drawPath` in `map.js` to have a single function that calculates the path when given a sector, x, y and speed.
- [ ] Write TCC function that calculates paths on a sector level. _Maybe make a table to speed up the process._
- [ ] Make simple DOM to test the TCC
- [ ] Make pretty DOM.
- [ ] Incorporate `storedPath` with the TCC

**Comment by Tsunder** [comment]
I think multi-part sectors (like Heze north/south, or the Core) would need to be solved to achieve this.... and that might be hard due to current map data setup?

**Comment by BartKieviet** [comment]
Yeah working on that now, splitting sectors is not that hard luckilly. Now working on the double WH connections (*e.g.* Bewaack - Miayda
